oneview_redfish_toolkit/authentication.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import logging
+import logging.config
+
+# 3rd party libs
+from flask import abort
+from flask_api import status
+from hpOneView import HPOneViewException
+from hpOneView.oneview_client import OneViewClient
+
+# Modules own libs
+from oneview_redfish_toolkit import config
+from oneview_redfish_toolkit import connection
+
+
+# Globals vars:
+#   globals()['map_tokens']
+
+def get_map_tokens():
+    return globals()['map_tokens']
+
+
+def set_map_tokens(new_map):
+    globals()['map_tokens'] = new_map
```

Comment:
Isn't better to assign a empty dict here rather than receive it as argument?

Reply:
Done. Maybe after multiple OneView support implementation we could move this initialization method to run at application startup.